lms/djangoapps/bookmarks/views.py

```diff
@@ -0,0 +1,278 @@
+"""
+For more information, see:
+https://openedx.atlassian.net/wiki/display/TNL/Bookmarks+API
+"""
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+from django.utils.translation import ugettext as _
+
+from rest_framework import status
+from rest_framework import permissions
+from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
+from rest_framework.generics import ListCreateAPIView
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey, UsageKey
+
+from bookmarks.serializers import BookmarkSerializer
```

Comment:
You can move this to the bottom of the imports along with the others from the same app.